### PR TITLE
Merge suitable rules changes from systemd

### PR DESCRIPTION
**Pull request by bbonev**
Merge up to commit 5674b74c4f99e433fd8e7242e9f16f6ddfece94c

- change @bindir@ to {{ROOTBINDIR}} in templates (better future tracking)
- keep rules/50-udev-default.rules as plain instead of template
- add
  60-autosuspend.rules
  60-fido-id.rules
  70-camera.rules
  70-memory.rules
  81-net-dhcp.rules

### CODE REVIEW

File: rules/64-btrfs.rules.in [hunk 1]
```diff
@@ -3,6 +3,7 @@
 SUBSYSTEM!="block", GOTO="btrfs_end"
 ACTION=="remove", GOTO="btrfs_end"
 ENV{ID_FS_TYPE}!="btrfs", GOTO="btrfs_end"
+ENV{SYSTEMD_READY}=="0", GOTO="btrfs_end"
```
Comment (ArsenArsen) [inline rules/64-btrfs.rules.in:6]: where does this come from?

**Review by ArsenArsen — APPROVED** [review]
save for that one question, this seems fine. I'm not aware of anything even exporting SYSTEMD_READY even

File: rules/64-btrfs.rules.in (hunk 1 above)
Comment (ArsenArsen) [reply rules/64-btrfs.rules.in:6]: I couldn't find it in sysd either, and I'm not aware of anything using it, which is why I ask

Comment (bbonev) [reply rules/64-btrfs.rules.in:6]: https://github.com/systemd/systemd/blob/main/rules.d/64-btrfs.rules.in#L6

Comment (ArsenArsen) [reply rules/64-btrfs.rules.in:6]: ah, huh, that slipped by my grepping. I am guessing this option has no impact on boot with btrfs? namely, I'm not sure whether that check fails or passes with SYSTEMD_READY unset

**Pull request merged**